exec/java-exec/src/main/java/org/apache/drill/exec/ops/FragmentContext.java

```diff
@@ -445,11 +447,16 @@ public PartitionExplorer getPartitionExplorer() {
   }
 
   @Override
-  public ValueHolder getConstantValueHolder(String value, Function<DrillBuf, ValueHolder> holderInitializer) {
-    ValueHolder valueHolder = constantValueHolderCache.get(value);
+  public ValueHolder getConstantValueHolder(String value, MinorType type, Function<DrillBuf, ValueHolder> holderInitializer) {
```

Comment:
it would be nice if we could share the common codes of getConstantValueHolder() in QueryContext/FragmentContext, and wrap them in a helper function. But this is optional.